phpBB/includes/constants.php

```diff
@@ -223,6 +223,11 @@
 // Additional constants
 define('VOTE_CONVERTED', 127);
 
+// Storage update methods
+define('STORAGE_UPDATE_TYPE_CONFIG', 0);
```

Comment:
Hm, I'm not a fan of adding new defines. If there are no other fitting places, I'd rather have a storage specific constants file that then defines these than adding more defines.